code/modules/mob/living/silicon/ai/ai_verbs.dm

```diff
@@ -226,6 +226,21 @@
 	popup.set_content(dat)
 	popup.open(FALSE)
 
+/mob/living/silicon/ai/verb/toggle_anchor()
```

Comment:
Don't add new verbs.

Make this an ability with a proper keybind, or something like that